### PR TITLE
Keep cosmosdb instance for travis ci test.

### PR DESCRIPTION
**Pull request by Incarnation-p-lee**
* Add profile for travis ci.

Signed-off-by: Pan Li <panli@microsoft.com>

### CODE REVIEW

**Comment by Incarnation-p-lee** [comment]
travis ci upgrade result in setup script got some warning/error message in key field. That may easy to filter out or fix the python issue from az command line. So I prefer to keep one instance for travis CI testing.

File: .travis.yml [hunk 1]
```diff
@@ -1,18 +1,16 @@
...
-      script:
-        - chmod +x install.sh && ./install.sh > /dev/null
-        - set -o pipefail
-        - mvn -P integration-test-azure clean cobertura:cobertura-integration-test | grep -v "DEBUG"
-        - bash <(curl -s https://codecov.io/bash)
-      if: type = pull_request OR (type = push AND branch = master)
+before_script:
+  - chmod +x ./src/libs/integration-test-setup.sh
+  - ./src/libs/integration-test-setup.sh > src/test/resources/application.properties
+  - cat src/test/resources/application.properties
```
Comment (sophiaso) [inline .travis.yml]: Wonder is it better not to cat the properties? Without revealing the key and secret.

File: .travis.yml [hunk 2]
```diff
@@ -1,18 +1,16 @@
 language: java
 jdk:
   - oraclejdk8
-sudo: required
 
-jobs:
-  include:
-    - stage: build
-      script: mvn clean package
-      if: type = push AND branch != master
```
Comment (sophiaso) [inline .travis.yml]: Why these branch tests are removed? We do not need to do different builds for different branches and webhook types?

File: .travis.yml (hunk 1 above)
Comment (Incarnation-p-lee) [reply .travis.yml]: I put this cat here just FYI, for make sure the secret key is protected correctly. We can remove this in .travis.yml. But I recommend to keep it.

File: .travis.yml (hunk 2 above)
Comment (Incarnation-p-lee) [reply .travis.yml]: the new script has covered that already.

File: .travis.yml [hunk 3]
```diff
@@ -1,18 +1,16 @@
...
-        - bash <(curl -s https://codecov.io/bash)
-      if: type = pull_request OR (type = push AND branch = master)
+before_script:
+  - chmod +x ./src/libs/integration-test-setup.sh
+  - ./src/libs/integration-test-setup.sh > src/test/resources/application.properties
+  - cat src/test/resources/application.properties
+
+script:
+  - set -o pipefail
+  - mvn -P travis-ci-test clean cobertura:cobertura-integration-test | grep -E "INFO|Tests"
```
Comment (sophiaso) [inline .travis.yml]: Afraid the `grep -E "INFO|Tests"` here may miss some logs.

Comment (Incarnation-p-lee) [reply .travis.yml]: due to the limitation of travisCI log 4MB, how about using `grep -v DEBUG` ?

**Pull request merged**